packages/css/src/components/password-input/README.md

```diff
@@ -0,0 +1,25 @@
+<!-- @license CC0-1.0 -->
+
+# Password Input
+
+Helps users enter a password.
+
+Use this component when the input requires sensitive information, like passwords or PINs.
+It ensures that the input is not readable by others who might be looking at the screen.
+The characters entered are hidden, represented by squares.
+
+Consider setting the following attributes:
+
+1. Allow the user’s password manager to automatically fill the password through `autocomplete="current-password"`.
+   When asking for a new password, use `autocomplete="new-password"` instead.
+2. Add a `minlength` attribute to ensure passwords meet a minimum length requirement.
```

Comment:
```suggestion
2. Add a `minlength` attribute to ensure passwords meet a minimum length requirement.
  Describe this in the [Field](?path=/docs/components-forms-field--docs)’s description as well.
```

Reply:
Updated in a commit – we don’t (need to) use the `path` param.